hi-ml-histopathology/testhisto/testhisto/models/test_deepmil.py

```diff
@@ -142,7 +151,7 @@ def validate_metric_inputs(scores: torch.Tensor, labels: torch.Tensor) -> None:
     def is_integral(x: torch.Tensor) -> bool:
         return (x == x.long()).all()  # type: ignore
 
-    assert scores.shape == labels.shape
+    assert scores.shape[0] == labels.shape[0]
```

Comment:
Related to my other comments, what would you think of writing instead
```python
assert labels.shape == (scores.shape[0],)
```
to enforce that `labels` is always a 1D tensor?